Gemfile

```diff
@@ -18,7 +18,7 @@ gem 'rack-cors'
 # Instance/Course settings
 gem 'settings_on_rails', git: 'https://github.com/bivanalhar/settings_on_rails'
 # Manage read/unread status
-gem 'unread'
+gem 'unread', '0.13.1'
```

Comment:
this one is purely oversight. I did this so that the `bundle install` command can fetch the latest version, but I did not revert it back once it's done. Will take note of this in the future not to leave it locked